Examples/MAX32572/SPIXR/main.c

```diff
@@ -161,9 +164,11 @@ int main(void)
 
     if (fail == 0) {
         printf("EXAMPLE SUCCEEDED\n");
+        return 0;
     } else {
         printf("EXAMPLE FAILED\n");
+        return -1;
     }
 
-    while (1) {}
+    return 0;
```

Comment:
Same as
https://github.com/Analog-Devices-MSDK/msdk/pull/317/files#r1063432996
Please apply it for all case in this PR